roles/ceph-common/tasks/main.yml

```diff
@@ -26,15 +26,15 @@
   when: ansible_os_family == 'Debian'
 
 - name: check for a ceph socket
-  shell: "stat /var/run/ceph/*.asok > /dev/null 2>&1"
+  stat: path=/var/run/ceph/*.asok
```

Comment:
Looks like this breaks the deployment in the end... :/ socket.rc can't be evaluated.
Same for rgw socket.